lib/chef/knife/ec2_server_create.rb

```diff
@@ -797,7 +794,11 @@ def bootstrap_common_params(bootstrap)
         bootstrap.config[:bootstrap_vault_item] = locate_config_value(:bootstrap_vault_item)
         bootstrap.config[:use_sudo_password] = locate_config_value(:use_sudo_password)
         bootstrap.config[:yes] = locate_config_value(:yes)
-        bootstrap.config[:tags] = config[:chef_tag] if locate_config_value(:chef_tag)
+        if locate_config_value(:chef_tag)
+          bootstrap.config[:tags] = config[:chef_tag]
```

Comment:
if I am not wrong we user tags options is tag the server. So why we assigning the chef_tag value to tags options.  And --chef-tag is used to tag the node in chef server ?

Reply:
Actually `tags` is used for tagging in both AWS and Chef so we need to assign `chef_tag` values in tags here if provided.